docs/content/reference/resource-schema/core-schema/environment-schema/index.md

```diff
@@ -56,7 +56,7 @@ Details on what to run and how to run it are defined in the `container` property
 |------|:--------:|-------------|---------|
 | \<recipe-name\> | y | The name of the Recipe. Must be unique within the resource-type. | `myrecipe`
 | templateKind | y | Format of the template provided by the recipe. Allowed values: bicep | `'bicep'`
-| templatePath | y | The path to the Recipe contents. For Bicep Recipes this is a Bicep module registry address. | `'mycr.azurecr.io/recipes/myrecipe:1.0'`
+| templatePath | y | The path to the Recipe contents. For Bicep Recipes this is a Bicep module registry address. | `'mycr.ghcr.io/recipes/myrecipe:1.0'`
```

Comment:
Need help, is this still a valid path example?

Reply:
```suggestion
| templatePath | y | The path to the Recipe contents. For Bicep Recipes this is a Bicep module registry address. | `'ghcr.io/USERNAME/recipes/myrecipe:1.0'`
```